bin/assigner/CMakeLists.txt

```diff
@@ -41,35 +41,18 @@ target_link_libraries(${CURRENT_PROJECT_NAME}
                       LLVMIRReader
                       LLVMCore
                       LLVMSupport
-                      crypto3::algebra
-                      crypto3::block
+                      crypto3::all
                       crypto3::blueprint
-                      crypto3::codec
-                      crypto3::math
-                      crypto3::multiprecision
-                      crypto3::pkpad
-                      crypto3::pubkey
-                      crypto3::random
-                      crypto3::zk
-
-                      crypto3::assigner
-                      crypto3::transpiler
-
-                      marshalling::core
-                      marshalling::crypto3_algebra
-                      marshalling::crypto3_multiprecision
-                      marshalling::crypto3_zk
-
                       ${Boost_LIBRARIES})
 
 target_include_directories(${CURRENT_PROJECT_NAME} PUBLIC
                            $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>
                            $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-                           ${CMAKE_CURRENT_SOURCE_DIR}/../../libs/circifier/llvm/include
-                           ${CMAKE_CURRENT_SOURCE_DIR}/../../libs/circifier/utils/bazel/llvm-project-overlay/llvm/include
+                           ${CMAKE_SOURCE_DIR}/libs/circifier/llvm/include
+                           ${CMAKE_SOURCE_DIR}/libs/circifier/utils/bazel/llvm-project-overlay/llvm/include
+                           ${CMAKE_SOURCE_DIR}/libs/assigner/include
```

Comment:
done

Reply:
`libs/assigner/include` still remains

Reply:
I've tried removing it, it seems, it does not export the targets the right way.

Reply:
Yes, it is not enough just to remove it. But you can restore `crypto3::assigner` and `crypto3::transpiler` as it was at lines 55-56 of the previous version of the file. It is basically the same way as you have just fixed `bin/recursive_gen/CMakeLists.txt`, so I am almost sure that it would work.

Reply:
Let's try use interace libraries: crypto3::assigner and crypto3::transpiler.
Sound like quick change. If doesn't works - keep as is